files/es/learn/javascript/howto/index.md

```diff
@@ -214,7 +214,7 @@ Para obtener más información sobre la depuración de JavaScript, consulte [Man
 
 - [¿Qué son los controladores de eventos y cómo los usa?](/es/docs/Learn/JavaScript/Building_blocks/Events#Event_handler_properties)
 - [¿Qué son los controladores de eventos en línea?](/es/docs/Learn/JavaScript/Building_blocks/Events#Inline_event_handlers_%E2%80%94_don%27t_use_these)
-- [¿Qué hace la función `addEventListener()`y cómo la usa?](</es/docs/Learn/JavaScript/Building_blocks/Events#addEventListener()_and_removeEventListener()>)
+- [¿Qué hace la función `addEventListener()` y cómo la usa?](</es/docs/Learn/JavaScript/Building_blocks/Events#addEventListener()_and_removeEventListener()>)
```

Comment:
the `<>` are needed when a URL has characters like `()` in the URL. Since those have been removed from most of the slugs and the new anchor generation, the can usually be removed as long as those characters are removed
```suggestion
- [¿Qué hace la función `addEventListener()` y cómo la usa?](/es/docs/Learn/JavaScript/Building_blocks/Events#addEventListener_and_removeEventListener)
```